packages/dart/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## [5.1.3](https://github.com/parse-community/Parse-SDK-Flutter/compare/dart-5.1.2...dart-5.1.3) (2023-05-29)
```

Comment:
Corrected the changelog date, assuming we release today.

@mbfakourii Could you make a formal review?

```suggestion
## [5.1.3](https://github.com/parse-community/Parse-SDK-Flutter/compare/dart-5.1.2...dart-5.1.3) (2023-07-18)
```

Reply:
I checked, there is no problem

Reply:
I mean, could you make a review on GitHub?

Reply:
sure